doc/modules/mod_pubsub.md

```diff
@@ -65,16 +65,8 @@ Only one nodetree can be used per host and is shared by all node plugins.
 
 #### `<<"tree">>`
 
-Stores nodes in a database (only mnesia supported).
-
-#### `<<"virtual">>`
-
-Does not store nodes in a database.
-This saves resources on systems when the number of nodes is large.
-When using the `<<"virtual">>` nodetree, you can only enable these node plugins: `[<<“flat”>>,<<“pep”>>]` or `[<<“flat”>>]`; any other plugin configuration will not work.
-Also, all nodes will have the defaut configuration, and this can not be changed.
-
-Using the `<<“virtual”>>` nodetree requires the pubsub module to be started with a clean database, it will not work if you used the default `<<“tree”>>` nodetree before.
+Stores nodes in a database (`mnesia` and `rdbms`).
```

Comment:
Stores nodes in a database (`mnesia` and `rdbms`).
or 
Stores nodes in a database (`mnesia` or `rdbms`).
??

Reply:
@goddammit ??

Reply:
Or 
"Stores nodes in a database (either `mnesia` or `rdbms`)" if I understand you correctly